build.gradle

```diff
@@ -41,14 +41,17 @@ repositories {
   mavenLocal()
   mavenCentral()
   maven {
+    url "https://maven.google.com"
```

Comment:
can be replace by google() ?

Reply:
I know, it depends which version you use and what dependencies. sometimes it's not an option to use google() so this can be changed in a future PR